charger/abl.go

```diff
@@ -31,24 +31,18 @@ import (
 type ABLeMH struct {
 	log  *util.Logger
 	conn *modbus.Connection
+	curr uint16
 }
 
 const (
 	ablRegFirmware   = 0x01
 	ablRegStatus     = 0x04
-	ablRegMode       = 0x05
 	ablRegAmpsConfig = 0x14
 	ablRegStatusLong = 0x2E
 
-	ablStatusDisabled = 0xE0
-
 	ablAmpsDisabled uint16 = 0x03E8
-	ablModeEnable   uint16 = 0xA1A1
-	ablModeDisable  uint16 = 0xE0E0
 )
 
-const ablStatusOutletDisabled = 0xE0
-
 var ablStatus = map[byte]string{
 	0xB1: "EV is asking for charging",
```

Comment:
Der Vollständigkeit halber würde hier noch
`0xA1: "Waiting for EV",`
fehlen

Reply:
Nope. Das wird direkt Staus `A`

Reply:
Aber dann Bx und Cx doch auch, oder?